scripts/variantstore/beta_docs/run-your-own-samples.md

```diff
@@ -27,22 +27,22 @@ For more information about reblocking, check out [WARP Whole Genome and Exome Pi
+|-------------------------------------------|-------------------------------------------------------------------------------------------|----------------------------|
+| Ref                                       | Reference allele.                                                                         | ---                        |
+| Alt                                       | Alternate allele.                                                                         | ---                        |
+| AS_RAW_MQ, RAW_MQandDP, or RAW_MQ         | RMS mapping quality (‘AS’: allele-specific).                                              | Required for VETS training |
 | AS_RAW_MQRankSum or Map_QUAL_RANK_SUM_KEY | Z-score from Wilcoxon rank sum test of alternate versus reference read mapping qualities. | Required for VETS training |
-| QUALapprox | Sum of PL[0] values; used to approximate the QUAL score. | Required for VETS training      |
-| AS_QUALapprox | Allele-specific sum of PL[0] values; used to approximate the QUAL score. | Required for VETS training      |
-| AS_SB_TABLE or STRAND_BIAS_BY_SAMPLE | Allele-specific forward/reverse read counts for strand bias tests. | Required for VETS training      |
-| AS_VarDP, VarDP, or DP | Depth over variant genotypes, or read depth  (‘AS’: allele-specific). | Required for VETS training      |
-| call_GT | Genotype. | ---                             |
-| call_GQ | Genotype quality. | ---                             |
+| QUALapprox                                | Sum of PL[0] values; used to approximate the QUAL score.                                  | Required for VETS training |
+| AS_QUALapprox                             | Allele-specific sum of PL[0] values; used to approximate the QUAL score.                  | Required for VETS training |
+| AS_SB_TABLE or STRAND_BIAS_BY_SAMPLE      | Allele-specific forward/reverse read counts for strand bias tests.                        | Required for VETS training |
+| AS_VarDP, VarDP, or DP                    | Depth over variant genotypes, or read depth  (‘AS’: allele-specific).                     | Required for VETS training |
+| call_GT                                   | Genotype.                                                                                 | ---                        |
+| call_GQ                                   | Genotype quality.                                                                         | ---                        |
 
 ### What does it return as output?
 
-The following files are stored in the workspace workflow execution bucket under Data>Files, or the Google bucket specified in the `extract_output_gcs_dir` input.
+The following files are stored in the workspace workflow execution bucket under Data>Files, or the location specified by the `output_gcs_dir` workflow output.
```

Comment:
Is this still true? Will it ever be under workflow execution bucket under Data>Files? Seems like because of this PR it will always go to `gs://<workspace bucket>/output_vcfs/by_submission_id/<submission id>` unless `extract_output_gcs_dir` is specified.